server/planning/content_api/events/__init__.py

```diff
@@ -8,23 +8,39 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
+from typing import Optional
 from superdesk.flask import g
+from pydantic import BaseModel
 from eve.utils import ParsedRequest
 from superdesk.core.module import Module
 from superdesk.core.web import EndpointGroup
 from werkzeug.datastructures import MultiDict
 from superdesk.core.types import Request, Response
 from .event import content_api_event_resource_config, ContentAPIEventService
+from planning.content_api.utils import ALLOWED_PARAMS
 
 event_endpoints = EndpointGroup("events_capi", __name__)
 
 
+class EventListParams(BaseModel):
```

Comment:
As these are common fields across both Events and Planning endpoints, it would be good to define these in common types (or at least a common base class with fields that are used with both endpoints).

Also, no need for `Optional[str] | None`, Python now supports this.
You can instead use `str | None = None`